### DISCUSSION TITLE
Feedback: Coverage phase

### DISCUSSION THREAD

**Discussion by Stranger6667** [schemathesis/schemathesis#2418] [Q&A]
Schemathesis primarily generates randomized tests; therefore, the outcomes can vary between test runs.
This feature enables a new test phase that aims to generate a set of examples that cover common testing patterns for each parameter or property.
Such tests include checking boundary values, omitting required properties, etc.

### How to Enable

CLI option: `--experimental=coverage-phase`

### Feedback

We're eager to hear your feedback:
- Did this change help in finding bugs in your APIs?
- Have you encountered any issues or unexpected behavior?
- Any additional feedback or suggestions?

**Comment by mheguy** [comment]
This has been immediately helpful. Thanks a bunch.
